nasa-gibs/worldview
# WV-2978 E2E Test Fixes

## Description
This fix adds a new url parameter `e2e` that can be added to urls, specifically to be used in e2e tests. The url parameter prevents the notification modal from opening on page load, which was causing inconsistency in tests passing due to variable time to load the modal onscreen.

## How To Test
1. `git checkout wv-2978-e2e-test-fixes`
2. `npm ci`
3. `npm run watch`
4. In `app.js`, change line 203 from `numberOutagesUnseen,` to `numberOutagesUnseen: notifications.number,` temporarily to force the notifications modal to open on page load.
5. Open another Terminal window to the repo directory and run `npx playwright test animation/animation-test.spec.js` and verify that the tests pass.
